tests/test_browser.py

```diff
@@ -2598,6 +2600,28 @@ def test_webgl2_garbage_free_entrypoints(self):
   def test_webgl2_backwards_compatibility_emulation(self):
     self.btest(path_from_root('tests', 'webgl2_backwards_compatibility_emulation.cpp'), args=['-s', 'USE_WEBGL2=1', '-s', 'WEBGL2_BACKWARDS_COMPATIBILITY_EMULATION=1'], expected='0')
 
+  @no_macos('WebGL 2.0 Compute is not enabled/supported')
+  @requires_graphics_hardware
+  def test_webgl2_compute(self):
+    if not (is_chrome() and '--enable-webgl2-compute-context' in EMTEST_BROWSER):
+      self.skipTest('WebGL 2.0 Compute is not enabled/supported')
+    # To skip the noisy frames produced at the beginning of ray tracing
+    create_test_file('pre.js', dedent('''\
+        window.requestAnimationFrame = (function() {
+          var counter = 0;
+          var realRequestAnimationFrame = window.requestAnimationFrame;
+          requestAnimationFrame = function(func) {
+            counter++;
+            if (counter == 2000) {
```

Comment:
how reliable is this number - does the test fail randomly now and then?

Reply:
Each frame of the test should be deterministic, so it's safe to use any value here.

This is the 10th frame:

![emsmallpten10](https://user-images.githubusercontent.com/1270848/55851924-65538700-5b8d-11e9-9c41-bd586fdcc5b3.png)

And this is the 2000th frame:

![emsmallpten2000](https://user-images.githubusercontent.com/1270848/55851941-756b6680-5b8d-11e9-879f-4f7fed6b6cb2.png)

The magic number 2000 is just for more pleasant double check with human eyes.

Reply:
Thanks, 2000 sounds good then.